utils/config.mli

```diff
@@ -163,6 +163,12 @@ val model: string
 val system: string
 (** Name of operating system for the native-code compiler *)
 
+val target_os_type: string
+(** Operating system targetted by the native-code compiler. One of
+-  ["Unix"] (for all Unix versions, including Linux and Mac OS X),
```

Comment:
```suggestion
-  ["Unix"] (for all Unix versions, including Linux and macOS),
```

Reply:
It could also be fixed in `Sys` by the way but I’ll leave that out of that PR.